File: backend/Worker/Models/PersonModel.cs
using AutoMapper;
using Global;
using Repository.Entities;
using System;
using System.Collections.Generic;

namespace Worker.Models
{
    [AutoMap(typeof(PersonEntity), ReverseMap = true)]
    [AutoMap(typeof(List<PersonEntity>), ReverseMap = true)]
    public class PersonModel
    {
        public Guid Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int Age { get; set; }
        public Gender Gender { get; set; }
        public SocialStatus SocialStatus { get; set; }
    }
}

File: frontend/src/app/dashboard/common/models/person.model.ts
import {Gender} from "../enums/gender.enum";
import {SocialStatus} from "../enums/social-status.enum";

export class PersonModel {
  public id?: string;
  public firstName!: string;
  public lastName!: string;
  public age!: number;
  public gender!: Gender;
  public socialStatus: SocialStatus;
}
